os/arch/arm/include/s5j/dma.h

```diff
@@ -21,7 +21,12 @@
 /****************************************************************************
  * Pre-processor Definitions
  ****************************************************************************/
-#define s5j_dmasetup(a, b) if ((b)->setup) { (b)->setup(a, b) };
+#define s5j_dmasetup(a, b)		\
+do {					\
+	if ((b)->setup) {		\
+		(b)->setup(a, b);	\
+	}				\
+} while(0)
```

Comment:
```
} while (0)
```
Be aware of coding rule first. That is basic. It is difficult to tell every time.

Reply:
Ok, I will take care from next time.